draft-ietf-rtcweb-jsep.xml

```diff
@@ -2445,6 +2445,12 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
 
             <t>The "a=rtcp-rsize" line MUST only be added if present in
             the most recent answer.</t>
+
+            <t>An "a=bundle-only" line MUST NOT be added, as indicated
+            in <xref target="I-D.ietf-mmusic-sdp-bundle-negotiation"/>,
+            Section 6. Instead, JSEP implementations simply omit ICE
+            parameters for m= lines which can only be used when bundled.
```

Comment:
Should the second sentence be phrased as a MUST? Also, would "for m= sections" be more accurate than "for m= lines"?